content/metadata.yaml

```diff
@@ -43,6 +43,17 @@ authors:
        - Lustgarten Foundation
        - Emerson Foundation
        - Allegheny Health Network
+  - github: vjcitn
+    name: Vincent J. Carey
+    initials: VJC
+    orcid: 0000-0003-4046-0063
+    email: stvjc@channing.harvard.edu
+    affiliations:
+      - Channing Division of Network Medicine, Brigham and Women's Hospital, Harvard Medical School
+    funders:
+       - National Institutes of Health, National Human Genome Research Institute
```

Comment:
Do you want to put grant numbers here? Not necessary but optional.

Reply:
Thanks, I didn't see that level of detail on other author data but would supply if appropriate.

Reply:
I see now that @ejfertig also didn't put a grant ID. I like to have the grant IDs there because I want whoever is entering it into the manuscript system to link my grant. It's definitely your call though!